### DISCUSSION TITLE
关于SSR Plus+打开油管速度慢转圈圈，无法运行怕死我跟helloword。

### DISCUSSION THREAD

**Discussion by kino516** [coolsnowwolf/lede#9583] [R68S / 电犀牛专区] [answered]
测试过DOCK版跟无DOCK版本6.10 6.11 6.15都是一样的问题。
描述：
1、用自带的SSRP代理添加订阅地址，选择节点启用正常。但是打开油管视频后，视频显示黑屏状态一直转圈，等待1分钟才开始播放视频。
2、使用软件包安装passwall+helloword。添加订阅地址，选择节点。运行后只有PDNSD显示正在运行。客户端+TCP+UDP都显示未运行。
PS:在朋友的R68S上测试过，他的又是正常的。我都怀疑是不是我的硬件有问题。

**Comment by coolsnowwolf** [comment, marked as answer]
关闭 Ipv6

![image](https://user-images.githubusercontent.com/31687149/173863686-5b12e96a-e4fa-4e95-9b74-f5b53d42cae3.png)
![image](https://user-images.githubusercontent.com/31687149/173863726-d93210f9-5015-4489-acaf-a968464af6c8.png)

**Reply by kino516** [reply]
关闭IPV6正常了。怕死我用不了测试了是我节点的问题。纯SS节点无法使用。
